src/guides/v2.4/comp-mgr/trouble/current-search-engine.md

```diff
@@ -0,0 +1,57 @@
+---
+group: software-update-guide
+title: Your current search engine is not supported
+functional_areas:
+  - Upgrade
+---
+
+The `Your current search engine, <Engine Name>, is not supported. You must install a supported search engine before upgrading. See the System Upgrade Guide for more information.` error message indicates that Magento version you are upgrading from is configured to use a catalog search engine that is not supported in the Magento version you are upgrading to. This error means one of the following conditions are true on the down-level version of Magento:
+
+*  The search engine is set to MySQL.
+*  The search engine is set to a version of Elasticsearch that is no longer supported.
+
+Use the following command to check the current search engine:
+
+```bash
+bin/magento config:show catalog/search/engine
+```
+The error occurs if the returned value is `mysql` or `elasticsearch`.
+
+{:.bs-callout-warning}
+If you have received this error, Magento is in an inconsistent state, and you cannot access the Admin. We recommend that you revert to your previous version of Magento while you resolve this error. To do this, run the `composer require magento/product-enterprise-edition=<version>` or `composer require magento/product-community-edition=<version>` command, where `<version>` is the version of Magento you were running **before** the upgrade. For example, `2.3.5`.
```

Comment:
Again, consider separating the commands from the paragraph to improve readability.